gnomad/assessment/validity_checks.py

```diff
@@ -251,7 +254,7 @@ def summarize_variant_filters(
     :param variant_filter_field: String of variant filtration used in the filters annotation on `ht` (e.g. RF, VQSR, AS_VQSR). Default is "RF".
     :param problematic_regions: List of regions considered problematic to run filter check in. Default is ["lcr", "segdup", "nonpar"].
     :param single_filter_count: If True, explode the Table's filter column and give a supplement total count of each filter. Default is False.
-    :param monoallelic_expr: Optional boolean expression of monoallelic status that logs how many monoallelic sites are in the Table.
+    :param monoallelic_expr: When passed, either a single boolean expression or dictionary of strings and boolean expressions to log how many monoallelic sites are in the Table. Default is None.
```

Comment:
```suggestion
    :param monoallelic_expr: Optional boolean expression or dictionary of strings and boolean expressions used to log how many monoallelic sites are in `t`.
```
minor rewording

Reply:
or I guess maybe `t` should be "Table" for consistency

Reply:
what about renaming monoallelic_expr if it's going to be used for monoallelic and only_het? I don't have a good suggestion though, the best I could think of is "unexpected_gt_expr"

Reply:
Yeah -- I was going to do something along those lines but figured thats a breaking change and choose slightly confusing over breaking change but there will probably be many breaking changes in the next release and this is pretty minor so `unexpected_gt_expr` works for me.

Reply:
`single_call_site_expr`?

Reply:
Though I guess that would limit the utility to these two cases

Reply:
good point about the breaking change, but I guess we have a few of those and will need to make a pypi release in the near future

what about `mono_call_site_expr`? is that too close to monoallelic